README.rst

```diff
@@ -351,6 +352,48 @@ mode (see **Using transparent mode** above) or for a specific request (see
 **Sending requests with automatically-mapped parameters** above), Zyte API
 parameters are chosen as follows by default:
 
+-   ``Request.url`` becomes ``url``, same as in requests with manually-defined
+    parameters.
+
+-   If ``Request.method`` is something other than ``"GET"``, it becomes
+    ``httpRequestMethod``.
+
+-   ``Request.headers`` become ``customHttpRequestHeaders``.
+
+-   ``Request.body`` becomes ``httpRequestBody``.
+
+-   If the COOKIES_ENABLED_ Scrapy setting is ``True`` (default), and unless
+    request metadata sets dont_merge_cookies_ to ``True`` as well:
```

Comment:
I wonder if cookie handling should make it opt-in. According to Zyte API docs, this is an experimental feature which can be dropped or changed at any time without releasing a new API version. Not sure though, the cons is that users won't get cookie handling by default.

Reply:
> The cons is that users won't get cookie handling by default.

We can warn about it the first time we get a requests where cookie would have been set, so I think it is reasonable to make it opt-in. On it…